scripts/quests/outlands/You_Call_That_a_Knife.lua

```diff
@@ -0,0 +1,217 @@
+-----------------------------------
+-- You Call That a Knife
+-----------------------------------
+-- Log ID: 5, Quest ID: 6
+-- Mhebi Juhbily : !pos 40 -11 -159 250
+-- Vah Keshura   : !pos 30 -8.5 -105 250
+-- Chef Nonberry : !pos -136 0.0 -91 159
+-- Pula Rhatti   : !pos  -18.5 -4 -38 250
+-----------------------------------
+local templeID = zones[xi.zone.TEMPLE_OF_UGGALEPIH]
+-----------------------------------
+
+---@type TQuest
+local quest = Quest:new(xi.questLog.OUTLANDS, xi.quest.id.outlands.YOU_CALL_THAT_A_KNIFE)
+
+quest.reward =
+{
+    fame = 75,
+    fameArea = xi.fameArea.WINDURST,
+    gil = 7200,
+    title = xi.title.YA_DONE_GOOD
+}
+
+local function isFightInProgress()
+    return GetMobByID(templeID.mob.COOK_OFFSET):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 1):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 2):isAlive() or
+        GetMobByID(templeID.mob.COOK_OFFSET + 3):isAlive()
+end
+
+local function confirmTradedItems(trade)
+    for i = 0, 7 do
+        trade:confirmSlot(i)
+    end
+end
```

Comment:
This could probably be moved to npcUtil since I imagine there must be other places we'd want to do this.

Reply:
Done ✅